API/MoodisAPI/apiMoodis/apiMoodis/Controllers/ImageInfoController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using apiMoodis.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using System.Data.Entity;
+
+namespace apiMoodis.Controllers
+{
+    public class ImageInfoController : ApiController
+    {
+        [HttpGet]
+        public IHttpActionResult GetAllImageInfos()
+        {
+            using (DatabaseContext dbContext = new DatabaseContext())
+            {
+                var imageInfos = dbContext.ImageInfos.Include(e => e.Emotions).ToList();
+                if (imageInfos.Count() == 0)
+                {
+                    return BadRequest("There are no ImageInfos");
+                }
+                else
+                {
+                    return Ok(imageInfos);
+                }
+            }
+        }
+
+        [HttpGet]
+        public IHttpActionResult GetImageInfoById(string id)
+        {
+            using (DatabaseContext dbContext = new DatabaseContext())
+            {
+                var entity = dbContext.ImageInfos.Single(ImageInfo => ImageInfo.Id == id);
```

Comment:
will it include emotions?

Reply:
Already answered it on one of your questions. Similar rules apply here